conftest.py

```
@@ -1094,6 +1094,11 @@ def prepared_plan(
                 "does not implement relink_shared_disks"
             )
 
+        if plan.get("disable_drs_for_vms", False) and not isinstance(clone_provider, VMWareProvider):
+            raise ValueError(
+                f"disable_drs_for_vms requires a VMware clone provider, got '{type(clone_provider).__name__}'"
+            )
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)